contribute/index.markdown

```diff
@@ -0,0 +1,11 @@
+---
+layout: std
+title: Enter the void - Contribute
```

Comment:
Void should have a capital V as it is a proper noun.

Reply:
Same issue again.

Reply:
Capitalised.